wranglerjs/index.js

```diff
@@ -28,6 +28,15 @@ if (args["no-webpack-config"] === "1") {
   config = require(join(process.cwd(), args["webpack-config"]));
 }
 
+if (config.target !== undefined && config.target !== "webworker") {
+  throw error(
+    "Building a Cloudflare Worker with target " + JSON.stringify(config.target) +
```

Comment:
Can we use string literals here? Also putting a stringified JSON in the middle of an error message seems... interesting. Do you have an example of what this would output?

Reply:
It's adding quotes arround string, normalize priting functions, NaN, undefined, etc.